_data/authors.yml

```diff
@@ -81,3 +81,6 @@ kmeht:
 jjergus:
   full_name: Ján
   fbid: 100001052236684
+
+daniel:
+  full_name: Daniel
```

Comment:
fbid might actually be needed

Reply:
As far as I can tell there's only 1 place it's used (picture on each blog post) and it's optional there.

I messaged Daniel to ask what name/FBID he wants, I'll update this before pushing.

Reply:
Let's just put my full name and fbid in here:

```
dneiter: 
  full_name: Daniel Neiter
  fbid: 1405460721 
```